models.go

```diff
@@ -1,3 +1,18 @@
 package main
 
 // Define your models here
+type user struct {
+	User_id    int    `json:"user_id"`
+	Name       string `json:"name"`
+	Password   string `json:"password"`
+	Email      string `json:"email"`
+	Salt       string `json:"salt"`
```

Comment:
I don't think we need to store salt in the db, since we already store the hashed password in our db.